html5-validation-engine.js

```diff
@@ -296,6 +305,17 @@
                     isNotValid = false,
                     type ="";
 
+                if (characterRestrictionPattern){
+                    var regex = new RegExp(characterRestrictionPattern);
+                    isNotValid = isNotValid || (!regex.test($input.val()) ? true : false);
```

Comment:
[RegExp.test()](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) returns a boolean already. The ternary is a little redundant in this case (!bool) ? true : false;